aries-site/src/themes/aries.js

```diff
@@ -51,6 +51,30 @@ export const aries = deepMerge(hpe, {
       xlarge: {}, // anything larger than 1440,
     },
   },
+  page: {
+    wide: {
+      width: {
+        min: '336px', // 336 + 24 (margin) + 24 (margin) = 384 (e.g. 'medium')
+        max: 'xxlarge', // 1536
+      },
+      // leave this for Design System site
+      xsmall: {
+        pad: { horizontal: 'large' },
+      },
+      small: {
+        pad: { horizontal: 'large' },
+      },
+      medium: {
+        pad: { horizontal: 'large' },
+      },
+      large: {
+        pad: { horizontal: 'xlarge' },
+      },
+      xlarge: {
+        pad: { horizontal: 'xlarge' },
+      },
```

Comment:
Are these nested too deep? They are currently a child of 'wide'. Shouldn't they be a child of 'page'?

Reply:
This is aligned with the current theme structure of Page. Padding is defined within page[kind][size] in addition to any other box props.